index.html

```diff
@@ -37,10 +37,22 @@
             background-image: url(images/email.svg);
         }
     </style>
+    <script type="application/ld+json">
+    {  
+       "@context":"http://schema.org",
+       "@type":"Organization",
+       "url":"https://ngplusdevelopment.com",
+       "logo":"https://ngplusdevelopment.com/images/logo.svg",
```

Comment:
:thinking: I don't think they are going to add a background to the logo. Do you have something that works on a white background?

Reply:
I don't yet but I will make a square one with a proper background, good tip!